utils.go

```diff
@@ -37,3 +39,13 @@ func stringIDToObjectID(id string) (primitive.ObjectID, error) {
 	}
 	return objectID, nil
 }
+
+// reads through the temperatures provided by lm-sensors
+// giving every device a different reading
+// on my laptop it provides 3 readings thus 3 devices
+func valueService(n int) float64 {
```

Comment:
This method feels too important to leave it in utils. Reading both, the comment and the implementation it looks like implementation of some interface that provides values for devices. It deserves designated place and interface. It also deserves to be replaceable since `lm-sensors` is not available everywhere).